docs/pages/badge.js

```diff
@@ -25,51 +24,255 @@ export default function BadgePage({ generatedDocGen }: {| generatedDocGen: DocGe
           <MainSection.Card
             cardSize="md"
             type="do"
-            title="When to Use"
+            title="When to use"
             description={`
           - Labeling and bringing awareness to a specific element or feature (e.g., something is new or required).
         `}
           />
           <MainSection.Card
             cardSize="md"
             type="don't"
-            title="When Not to Use"
+            title="When not to use"
             description={`
           - Providing feedback at the element level (e.g., displaying error messages). Use inline text instead.
           - Requiring interaction from users since Badges are always static and non-interactive.
         `}
           />
         </MainSection.Subsection>
       </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Align the badge to the top of large accompanying text. Center align for standard size text."
```

Comment:
Instead  of _standard size text_ , could we specify sizes?

Reply:
Maybe, I'd add a large Text  and a small text example in the same example. To exemplify both.